skyportal/handlers/api/sharing.py

```diff
@@ -113,20 +113,22 @@ def post(self):
                 for group in groups:
                     spec.groups.append(group)
                 # Grab obj_id for use in websocket message below
-                spec_obj_ids.append(spec.obj_id)
+                spec_obj_ids.append(spec.obj.internal_key)
 
         self.verify_and_commit()
 
-        spec_obj_ids = set(spec_obj_ids)
         phot_obj_ids = set(phot_obj_ids)
+        spec_obj_ids = set(spec_obj_ids)
```

Comment:
Should we note in the variable name that these are all internal?

Reply:
yes good point, let me do that

Reply:
@mcoughlin done